### ISSUE TITLE
Building on OSX

### ISSUE THREAD

**Issue by aquiles2k** [mxe/mxe#1281]
Hi,
i recently tried to build a fresh mxe with gcc on Mac OSX 10.11.3 (El Capitan) with MacPorts which failed at two points.

According to the index.html on mxe.cc i installed the following ports:
```
sudo port install \
    autoconf automake bison coreutils flex gettext \
    gdk-pixbuf2 glib2 gsed intltool libffi libtool \
    openssl p5-xml-parser p7zip scons wget xz
```

The first error was:
```
Failed to build package mxe-conf for target x86_64-apple-darwin15.3.0!
------------------------------------------------------------
checking for autoconf... /opt/local/bin/autoconf
checking for automake... /opt/local/bin/automake
checking for libtool... /usr/bin/libtool
./configure: line 1787: syntax error near unexpected token `0.16'
./configure: line 1787: `PKG_PROG_PKG_CONFIG(0.16)'
make[1]: *** [build-only-mxe-conf_x86_64-apple-darwin15.3.0] Error 2
real	0m7.433s
user	0m5.266s
sys	0m0.579s
------------------------------------------------------------
```
I fixed that by also installing the pkgconfig port

After that i got:
```
Failed to build package glib for target i686-w64-mingw32.shared!
------------------------------------------------------------
m4macros/glib-gettext.m4:39: error: m4_copy: won't overwrite defined macro: glib_DEFUN
m4macros/glib-gettext.m4:39: the top level
autom4te: /usr/bin/m4 failed with exit status: 1
aclocal: error: echo failed with exit status: 1
autoreconf: /opt/local/bin/aclocal failed with exit status: 1
make[1]: *** [build-only-glib_i686-w64-mingw32.shared] Error 1
real	0m11.390s
user	0m6.619s
sys	0m0.977s
------------------------------------------------------------
```

This seems to be a issue to other on MLs as well and most say to delete the glib-gettext.m4. I fixed it by patching the file to use `m4_copy_force` instead of `m4_copy` on both occurences on line 39 and 40.

**Comment by starius** [comment]
I have opened a pull request fixing both problems: https://github.com/mxe/mxe/pull/1283
Does it work for you?

**Comment by pavelvat** [comment]
The same problem for Linux:
http://pastebin.com/70tAVcdY

> Does it work for you?

Fixed in #1283 for Linux:
https://gist.github.com/pavelvat/3c17c638b0bf2d8739cb5b9bbcdc2a4a
